server/interceptor/interceptor.go

```diff
@@ -403,6 +403,8 @@ func newRPCInfo(req interface{}, quotaUser string) (*rpcInfo, error) {
 		switch req := req.(type) {
 		case logLeavesRequest:
 			info.tokens = len(req.GetLeaves())
+		case logLeavesRangeRequest:
+			info.tokens = int(req.GetCount())
```

Comment:
It's an int in the request proto. Someone mischievous could set it -ve though not sure it would break anything.

Reply:
Heh.  Added a >0 check.